docs/en/connector-v2/source/Hive-jdbc.md

```diff
@@ -0,0 +1,160 @@
+# Hive
+
+> JDBC Hive Source Connector
+
+## Support Those Engines
+
+> Spark<br/>
+> Flink<br/>
+> SeaTunnel Zeta<br/>
```

Comment:
Add Supported Versions

Reply:
Definitely supports 3.1.3 and 3.1.2, other versions need to be tested